### PR TITLE
[Android] Fix issue with XWalkView::setBackgroundColor

### PR DESCRIPTION
**Pull request by axinging**
setBackgroundColor doesn't work before page loaded. This fix enabled setBackgroundColor for XWalkView.

BUG=XWALK-1910

### CODE REVIEW

File: runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java [hunk 1]
```diff
@@ -782,9 +787,18 @@ public void setResourceClient(XWalkResourceClientInternal client) {
      */
     @Override
     @XWalkAPI
-    public void setBackgroundColor(int color) {
+    public void setBackgroundColor(final int color) {
```
Comment (halton) [inline runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java]: Why add `final` here? I do not see android.view.View.setBackgroundColor has this change.

File: runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java [hunk 2]
```diff
@@ -782,9 +787,18 @@ public void setResourceClient(XWalkResourceClientInternal client) {
...
         if (mContent == null) return;
         checkThreadSafety();
+        if (mIsLoaded == false) {
```
Comment (halton) [inline runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java]: nit, `if(!mIsLoaded)`

File: runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java (hunk 1 above)
Comment (axinging) [inline runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java]: Java require final if two method see the same variable.

Comment (halton) [inline runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java]: Okay, i see `src/android_webview/glue/java/src/com/android/webview/chromium/WebViewChromium.java` did the same thing.

File: runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java (hunk 2 above)
Comment (halton) [inline runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java]: Ignore this comment after talk with @axinging

**Comment by halton** [comment]
LGTM

**Comment by axinging** [comment]
@lincsoon

**Comment by axinging** [comment]
@lincsoon Please do not merge this. I will refine this to XWalkContent instead of XWalkViewInternal.

**Comment by axinging** [comment]
@lincsoon @halton, refine change from  XWalkViewInternal to XWalkContent. XWalkContent has code to handle Runnable before. So put together will be better.

**Comment by halton** [comment]
LGTM for new patch.

**Comment by lincsoon** [comment]
LGTM

**Comment by halton** [comment]
ping @kenchris @darktears for this PR got LGTM.

**Comment by axinging** [comment]
@kenchris @darktears

**Pull request merged**